runtime/bcverify/rtverify.c

```diff
@@ -307,7 +307,7 @@ matchStack(J9BytecodeVerificationData * verifyData, J9BranchTargetStack *liveSta
 
 	/* Jazz103: 120689 */
 	/* Target stack frame flag needs to be subset of ours. See JVM sepc 4.10.1.4 */
-	if (liveStack->uninitializedThis && !targetStack->uninitializedThis) {
+	if (!verifyData->createdStackMap && liveStack->uninitializedThis && !targetStack->uninitializedThis) {
```

Comment:
This should have a comment in the code explaining why it's not applicable to the createdStackMap case.

The code may end up looking cleaner if written in this form once the comment is added:
```
/* Long comment describing
 * why this check isn't applicable to
 * createdstackMaps
 */
if (!verifyData->createdStackMap) {
    /* Target stack frame flag needs to be subset of ours. See JVM sepc 4.10.1.4 */
    if (liveStack->uninitializedThis && !targetStack->uninitializedThis) {
        .....
    }
}
```

Reply:
Already updated the code & comment against the suggestion above.